src/test/java/com/linkedin/kmf/KafkaMonitorTest.java

```diff
@@ -14,8 +14,6 @@
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicInteger;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertFalse;
```

Comment:
Why get rid of these?

Reply:
this would remove usages of static imports in the Kafka Monitor Test class.